lib/utils.js

```diff
@@ -259,8 +263,8 @@ const sessionDataDefaultsFile = path.join(projectDir, 'app', 'data', 'session-da
 try {
   sessionDataDefaults = require(sessionDataDefaultsFile)
 } catch (e) {
-  console.error('Could not load the session data defaults from app/data/session-data-defaults.js')
-  console.error(e)
+  // console.error('Could not load the session data defaults from app/data/session-data-defaults.js')
+  // console.error(e)
```

Comment:
I'd suggest just deleting these lines if we don't want them.

```suggestion
  // do nothing
```

Reply:
I've rewritten this with a default value in the catch.